frontend/src/components/TodoTask/IconButton/IconButton.tsx

```diff
@@ -0,0 +1,35 @@
+import { Button } from 'antd';
+import { StarFilled, StarOutlined, BorderOutlined, CheckSquareOutlined } from '@ant-design/icons';
+import { blue } from '@ant-design/colors';
```

Comment:
A zaimportujmy proszę kolor z Theme w styled components :)
W naszym projekcie znajdziesz pod:
`Assets/styles/`
Kolor możesz ustawić własny :)

Reply:
zmieniłem to